resources/views/admin/layouts/partials/_navbar.blade.php

```diff
@@ -7,7 +7,7 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="index.html">ADMIN</a>
+                <a class="navbar-brand" href="index.html"><span style="color:green;">Borrowing Book Admin</span></a>
```

Comment:
[rank A] We need define all labels or texts in language files. How do you think?